services/webos-service-remote/sshbus.js

```diff
@@ -0,0 +1,73 @@
+import EventEmitter from 'events';
+import { readFileSync } from 'fs';
+import { Client } from 'ssh2';
+
+function getSshPrivKey() {
+    return readFileSync('/var/luna/preferences/webos_rsa');
+}
+
+function getSshPassphrase() {
+    try {
+        return readFileSync('/var/luna/preferences/webosbrew_devmode_passphrase').toString();
+    } catch (e) {
+        throw 'Dev Mode SSH passphrase not set';
+    }
+}
+
+function constructResponse(payload, isSubscription) {
+    return {
+        applicationID: () => '',
```

Comment:
While I see which (internal) API this might be inspired by, I am not entirely sure what would be the reason to pass an object full of callables instead of a plain object here :D

Reply:
This is copied directly from webos-service implementation. Since it's an internal API, I think maybe we can change this part.